torchbenchmark/util/framework/huggingface/model_factory.py

```diff
@@ -86,7 +87,7 @@ def __init__(self, name, test, device, batch_size=None, extra_args=[]):
             config.num_buckets = 128
         class_ctor = getattr(transformers, class_models[name][3])
         kwargs = {}
-        if name == "hf_Falcon_7b" or name == "hf_MPT_7b_instruct" or name == "phi_1_5":
+        if name == "hf_Falcon_7b" or name == "hf_MPT_7b_instruct" or name == "phi_1_5" or name == "hf_Yi":
```

Comment:
For better readability, let's have a list containing all the models require remote code and test if `name in remote_code_required`.